### PR TITLE
Implemented SpinnerWrapper component

### PR DESCRIPTION
**Pull request by csong202**
Bugs:
- spinner disappears on canvas when use_leaf_spinner is switched from true to false
 need to go to docs then back to canvas to see the spinner.  
- adjusting the scale of non-leaf spinner doesn't work on canvas
 need to go to docs then back to canvas to see the effect of changing the scale.  
- box for spinner in docs is too small

### CODE REVIEW

**Comment by csong202** [comment]
just realized that I need multiple components for different formats and sizes

File: client/src/components/SpinnerWrapper/SpinnerWrapper.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import React from "react";
 
 import { Spinner, spinner_configs } from "src/core/Spinner.js";
 
-import { LeafSpinner } from "./LeafSpinner.js";
+// eslint-disable-next-line no-restricted-imports
+import { LeafSpinner } from "../LeafSpinner.js";
```
Comment (ktw1016) [inline client/src/components/SpinnerWrapper/SpinnerWrapper.js:7]: I think it's fair to rename folder name `SpinnerWrapper` to `Spinner` and include LeafSpinner all together !

File: client/src/components/SpinnerWrapper/SpinnerWrapper.stories.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { SpinnerWrapper } from "./SpinnerWrapper.js";
+
+export default {
+  title: "SpinnerWrapper",
+  component: SpinnerWrapper,
+};
+
+const Template = (args) => <SpinnerWrapper {...args} />;
+
+export const Basic = Template.bind({});
```
Comment (ktw1016) [inline client/src/components/SpinnerWrapper/SpinnerWrapper.stories.js:12]: You should make a story for each spinner config found in `LeafSpinner.js`, `leaf_spinner_configs`!

**Review by ktw1016 — CHANGES_REQUESTED** [review]

**Comment by csong202** [comment]
Making a new branch (spinner_wrapper_story_v2) off of master and doing a pull request from there.

**Pull request closed without merging**